lib/nanoc/base/repos/store.rb

```diff
@@ -34,6 +34,12 @@ def initialize(filename, version)
       @version  = version
     end
 
+    # Logic for building tmp path from active environment and store name
+    # @api private
+    def self.tmp_path_for(env, store)
```

Comment:
The name `store` is confusing, as it is the _name_ of a store rather than the store itself. Rename to `store_name`.